src/main/resources/logger.properties

```diff
@@ -0,0 +1 @@
+logger.namespace=kafka-error-consumer
```

Comment:
Shouldn't this be in the configs? The same namespace will be used for all instances of this error consumer - logs will be difficult to interpret when we have multiple error consumers for multiple topics.